package/Cargo.toml

```diff
@@ -33,5 +33,9 @@ derive_more = { version = "2.0", features = ["from"] }
 proptest = { workspace = true, optional = true }
 proptest-derive = { workspace = true, optional = true }
 serde = { workspace = true, optional = true }
+serde_json = { workspace = true, optional = true }
```

Comment:
I think we want to move this to `dev-dependencies` and make it non-optional there